code/datums/gamemode/dynamic/dynamic_rulesets_latejoin.dm

```diff
@@ -123,8 +123,8 @@
 	enemy_jobs = list("Security Officer","Detective", "Warden", "Head of Security", "Captain")
 	required_enemies = list(2,2,1,1,1,1,1,0,0,0)
 	required_candidates = 1
-	weight = 4
-	cost = 10
+	weight = 3
+	cost = 30
```

Comment:
tripling the threat cost? 1 ninja now costs as much as a whole cult?

Reply:
Fair, maybe 20 is better? In line with wizards. Or 15? 18?

Reply:
hard to pin down, I guess it's up for discussion, 20 at least feels more reasonable

Reply:
I want to say 30 is enough

Reply:
20, done